builtin/sampled_dataset.h

```diff
@@ -0,0 +1,66 @@
+/** sampled_dataset.h                                               -*- C++ -*-
+    Francois Maillet, 11 janvier 2016
+    This file is part of MLDB. Copyright 2016 Datacratic. All rights reserved.
+
+*/
+
+#pragma once
+
+#include "mldb/core/dataset.h"
+#include "mldb/types/value_description.h"
+
+namespace Datacratic {
+namespace MLDB {
+
+
+/*****************************************************************************/
+/* SAMPLED DATASET CONFIG                                                    */
+/*****************************************************************************/
+        
+struct SampledDatasetConfig {
+
+    SampledDatasetConfig();
+
+    PolyConfigT<const Dataset> dataset;
```

Comment:
I would put a std::shared_ptr<TableExpression>.  This way one can simply put a string in the config instead of a json object.